packages/host/app/lib/current-run.ts

```diff
@@ -419,11 +419,18 @@ export class CurrentRun {
         },
       }) as SingleCardDocument;
       searchData = await api.searchDoc(card);
+
       if (!searchData) {
         throw new Error(
           `bug: could not derive search doc for instance ${instanceURL.href}`,
         );
       }
+
+      if (cardType.displayName === 'Card') {
+        searchData.cardType = cardType.name;
+      } else {
+        searchData.cardType = cardType.displayName;
```

Comment:
`cardType` is added to `searchData` so we can sort by it in `SearchIndex`'s `search` method